foreign_cc/built_tools/make_build.bzl

```diff
@@ -74,6 +75,25 @@ def _make_tool_impl(ctx):
         if os_name(ctx) == "macos":
             non_sysroot_ldflags += ["-undefined", "error"]
 
+        configure_options = [
+            "--without-guile",
+            "--with-guile=no",
+            "--disable-dependency-tracking",
+            "--prefix=$$INSTALLDIR$$",
+        ]
+
+        install_cmd = ["./make install"]
+
+        xcompile_options = detect_xcompile(ctx)
```

Comment:
This all looks correct but when does the built make tool (or pkgconfig?) ever get cross compiled? I would have thought that it was always in the `exec` configuration and so would always be built natively?

Reply:
In rfcc that's true, but due to annoying historical reasons, we build make for both the exec and the target. For autoconf, anything that doesn't do the --host thing when cross-compiling causes it to just break, so I defaulted to adding it everywhere that used autotools (we don't build pkg-config for the target, I was just being consistent to minimize surprise)